content/kubermatic/master/installation/install_kubermatic/_index.en.md

```diff
@@ -143,6 +143,9 @@ dex:
     userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
 
 kubermaticOperator:
+  image:
+    # image id for EE version
+    repository: "quay.io/kubermatic/kubermatic-ee"
```

Comment:
So if I follow the installation documentation like this, I cannot install the CE version. Can we make it more clear that this is an option **if** the user wants the EE?

Reply:
how does the config look like for the CE version? 
is it `docker pull quay.io/kubermatic/kubermatic` as default?

Reply:
Yep, for the CE version you don't need to set `kubermaticOperator.image.repository` at all, as the default value already points to the CE repo.